steps/github-step-repositories-describe/README.md

```diff
@@ -0,0 +1,6 @@
+# github-step-repositories-describe
+
+This step will describe the repositories belonging to an organization. It requires a
+token with the proper access, otherwise you'll very quickly hit GitHub API rate-limiting.
+
+See [spec.schema.json](spec.schema.json) for parameters.
```

Comment:
Our frontend will automatically pull this in so you don't need to explicitly link to it in the README! :)